documentation/branch-strategy.md

```diff
@@ -1,20 +1,20 @@
 # branch strategy and naming
 
 ## workflow
-Every major.minor has it's own develop branch named feature/major.minor/dev.
+Every major.minor has it's own main branch named feature/major.minor/main.
 
-Start by creating a feature branch from that develop branch and naming it feature/major.minor/some-descriptive-name. 
+Start by creating a feature branch from that main branch and naming it feature/major.minor/some-descriptive-name. 
 
-When ready for review, do a pull request to the feature/major.minor/dev branch.
+When ready for review, do a pull request to the feature/major.minor/main branch.
 
-When ready for release, merge from feature/major.minor/dev to release/major.minor branch.
+When ready for release, merge from feature/major.minor/main to release/major.minor branch.
 
 *Note: Keep in mind when starting to develop that merging may be necessary from previous branches.*
 
 ## examples
-- feature/1.2/dev develop-branch for 1.2 produces 1.2-beta.buildnumber
-- feature/1.2/a-task-to-solve for 1.2 produces 1.2-alpha.nomalized_branchname.buildnumber
-- release/1.2 for 1.2 produces 1.2.buildnumber (used as patch)
+- feature/1.2/main main-branch for 1.2 produces 1.2.patchnumber-beta.buildnumber i.e. 1.2.4-beta.5
```

Comment:
I think the "defacto" format is: `1.2.<patch-number>-beta.<build-number>`, to emphasise the "dynamic" parts.